website/src/blog/task-in-2023.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Introducing Experiments
+description: A look at where task is, where it's going and how we're going to get there.
+author: pd93
+date: 2024-05-09
+outline: deep
+---
+
+# Introducing Experiments
+
+<AuthorCard :author="$frontmatter.author" />
+
+Lately, Task has been growing extremely quickly and I've found myself thinking a lot about the future of the project and how we continue to evolve and grow. I'm not much of a writer, but I think one of the things we could do better is to communicate these kinds of thoughts to the community. So, with that in mind, this is the first (hopefully of many) blog posts talking about Task and what we're up to.
+
+## 📅 So, what have we been up to?
```

Comment:
Was there any particular reason to stop using shortcodes here? It seems like [they are supported](https://github.com/markdown-it/markdown-it-emoji/blob/master/lib/data/full.mjs). I'm generally not a fan of having emojis in code as they often render strangely and break the monospacing.

Same applies to other files.